src/main/java/com/andmcadams/wikisync/dps/WebSocketManager.java

```diff
@@ -48,11 +48,7 @@ public class WebSocketManager implements WSHandler
 	@Inject
 	private ClientThread clientThread;
 
-	private static final ExecutorService executorService = Executors.newSingleThreadExecutor(r -> {
-		Thread t = new Thread(r);
-		t.setDaemon(true);
-		return t;
-	});
+	private static final ExecutorService executorService = Executors.newSingleThreadExecutor(new ThreadFactoryBuilder().setDaemon(true).setNameFormat("wikisync-dps-thread-%d").build());
```

Comment:
"WikiSync DPS Manager"?

Reply:
Sure! I'm not used to Java styles - is it more common to do "wikisync-dps-manager" or "WikiSync DPS Manager"?